fbchat/client.py

```diff
@@ -467,8 +467,16 @@ def resetDefaultThread(self):
 
     """
     END DEFAULT THREAD METHODS
-    """
-
+    """ 
+    def removeFriend(self, friend_id=None):
+        """param friend_id: The id of the friend that you want to remove"""
```

Comment:
Perhaps you could add a short description of the method, along with a documentation of the return value?

Reply:
Yes, I will add it soon.